elastic/client.go

```diff
@@ -34,7 +37,16 @@ func NewClient(conf *ClientConfig) *Client {
 	c.User = conf.User
 	c.Password = conf.Password
 
-	c.c = &http.Client{}
+	if conf.Https {
+		c.Protocol = "https"
+		tr := &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
do we need a certificate?

Reply:
@siddontang nope, we don't need a certificate.